codebuild/integ_codebuild.config

```diff
@@ -68,6 +68,45 @@ source_clonedepth: 1
 source_version:
 env: TESTS=integration BUILD_S2N=true
 
+# ARM + OpenSSL111 + GCC6 + Corked and notCorked + Gcc4.8
+[CodeBuild:s2nARMIntegrationOpenSSL111Plus]
+image : aws/codebuild/standard:2.0
```

Comment:
[AWS Docs](https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-available.html) say only `aws/codebuild/amazonlinux2-aarch64-standard:1.0` is setup for arm (does this work?), or possibly a custom built image.

Reply:
I should have made this a draft PR, sorry!

I'm testing this out. Currently running into role issues in my current account. This won't be sustainable just yet because troposphere hasn't merged ARM_CONTAINER support just yet. We need them to support ARM before we can automate creating codebuild stages for ARM.

Reply:
linking...https://github.com/cloudtools/troposphere/pull/1699